modules/oe_content_persistent/src/Plugin/Filter/FilterPurl.php

```diff
@@ -109,11 +109,21 @@ public function process($text, $langcode) {
           continue;
         }
 
+        // If an anchor is set on the source url, we extract it here.
+        $anchor = '';
+        if (strpos($href, '#') !== FALSE) {
+          $explodedHref = explode('#', $href);
+          if (isset($explodedHref[1]) && !empty($explodedHref[1])) {
```

Comment:
I think isset can be replaced by empty, but I know I had some issues with this in the past. So for safety I included both.